bson/src/main/org/bson/BsonDocument.java

```diff
@@ -81,13 +82,25 @@ public BsonDocument(final List<BsonElement> bsonElements) {
      * @param value the value
      */
     public BsonDocument(final String key, final BsonValue value) {
+        this(1);
```

Comment:
It's quite common usage, especially in the driver itself, to do this:

     var doc = new BsonDocument(key, val).append(key2, val2).append(key3, val3)...
     
instead of      

     var doc = new BsonDocument().append(key, val).append(key2, val2).append(key3, val3)...

Given that, this change could do more harm than good.  Can you revert it?